src/test/java/seedu/address/model/entitylist/MentorListTest.java

```diff
@@ -0,0 +1,111 @@
+package seedu.address.model.entitylist;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+
+import seedu.address.commons.exceptions.AlfredModelException;
+import seedu.address.model.entity.Id;
+import seedu.address.model.entity.Mentor;
+import seedu.address.model.entity.Name;
+import seedu.address.model.entity.PrefixType;
+import seedu.address.testutil.TypicalMentors;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
This one I'm just curious as to what it does lol

Reply:
This is needed so I can run the @BeforeAll signature to make sure that the list is set to 0 before the start of each test if not my assert statements might gg 💣

Reply:
oh..not sure if i understand but cool lol

Reply:
BeforeAll is to signal that the function is a setup method that should be run before all tests. Quite useful. But javafx was complaining that I had to put this `@TestInstance` thing to use BeforeAll so i added it there